app/src/components/inputs/inputTagsSearch/inputTagsSearch.jsx

```diff
@@ -0,0 +1,152 @@
+    return (<div
+      className={cx('select2-item')}
+      key={option.value}
+      onClick={() => { selectValue(option); }}
+    >
+      <span>{option.label}</span>
+    </div>);
+  }
+  state = {
+    value: this.props.values,
+    searchPromptText: this.props.nothingFound,
+  };
+  onChange(value) {
+    this.setState({ value });
+    this.props.selectItem(value);
+  }
+  onInputChange(input) {
+    const diff = this.props.minLength - input.length;
+    if (this.props.dynamicSearchPromptText && this.props.minLength && diff > 0) {
+      const dynamicSearchPromptText = <FormattedMessage
+        id={'InputTagsSearch.dynamicSearchPromptText'}
+        defaultMessage={'Please enter {length} or more characters'}
+        values={{ length: diff }}
+      />;
+      this.setState({ searchPromptText: dynamicSearchPromptText });
+    } else {
+      this.setState({ searchPromptText: this.props.nothingFound });
+    }
+    return input;
+  }
+  getItems(input, uri) {
+    if (input.length >= this.props.minLength) {
+      return fetch(`${uri}${input}`)
+        .then((response) => {
+          const options = this.props.makeOptions(response);
+          return ({ options });
+        });
+    }
+    return Promise.resolve({ options: [] });
+  }
+  isValidNewOption({ label }, minLength, validation, isValidNewOption) {
+    return isValidNewOption.call(this, label, minLength, validation);
+  }
+  newItemLabel(label) {
+    if (this.props.showNewLabel) {
+      return (
+        <div>
+          <span>{label}</span>
+          <span className="new">New</span>
+        </div>
+      );
+    }
+    return label;
+  }
+  render() {
+    let SelectComponent;
+    if (this.props.isAsync) {
+      if (this.props.isCreatable) {
+        SelectComponent = AsyncCreatable;
+      } else {
+        SelectComponent = Async;
+      }
+    } else if (this.props.isCreatable) {
+      SelectComponent = Creatable;
+    } else {
+      SelectComponent = Select;
+    }
+    return (
+      <div className={cx('select-container')}>
+        <SelectComponent
+          loadOptions={input => this.getItems(input, this.props.uri)}
+          autoload={false}
+          cache={false}
+          className={cx('select2-search-tags')}
+          noResultsText={this.state.searchPromptText}
+          loadingPlaceholder={this.props.loadingPlaceholder}
+          searchPromptText={this.props.focusPlaceholder}
+          value={this.state.value}
+          options={this.props.options}
+          onInputChange={input => this.onInputChange(input)}
+          onChange={(value) => { this.onChange(value); }}
+          multi={this.props.multi}
+          optionRenderer={option => InputTagsSearch.optionRenderer(option, this.onChange)}
+          isValidNewOption={label => this.isValidNewOption(label, this.props.minLength,
+          this.props.validation, this.props.isValidNewOption)}
+          menuRenderer={params => InputTagsSearch.renderItems(params)}
```

Comment:
`menuRenderer={renderItems}`
(`renderItems` doesn't have to be a static class method)